amd_openvx_extensions/CMakeLists.txt

```diff
@@ -85,13 +85,18 @@ if(GPU_SUPPORT)
 endif(GPU_SUPPORT)
 
 if (NEURAL_NET)
-    if(GPU_SUPPORT AND OpenCL_FOUND AND BUILD_OPENCL AND miopengemm_FOUND AND miopen_FOUND)
-        add_subdirectory(amd_nn)
-        message("-- ${Green}AMD OpenVX Neural Network Extension -- amd_nn module added${ColourReset}")
-    elseif("${BACKEND}" STREQUAL "HIP" OR "${BACKEND}" STREQUAL "CPU")
-        message("-- ${Yellow}AMD OpenVX Neural Network Extension -- amd_nn module excluded with HIP/CPU Backend${ColourReset}")
+    if(GPU_SUPPORT AND miopengemm_FOUND AND miopen_FOUND)
+        if (OpenCL_FOUND AND BUILD_OPENCL)
+            add_subdirectory(amd_nn)
+            message("-- ${Green}AMD OpenVX Neural Network Extension -- amd_nn module added${ColourReset}")
```

Comment:
indicate the backend(HIP/OpenCL) in the message

Reply:
done.